### PR TITLE
Minor modifications

### PR DESCRIPTION
**Pull request by Christina96**
Minor modifications in chapters 1-4

### CODE REVIEW

File: manuscript/1-SoftReq.txt [hunk 1]
```diff
@@ -235,14 +235,14 @@
...
-πάντα τους χρήστες / διαχειριστές και τους πελάτες (που δεν χρειάζεται
+πάντα τους χρήστες/ διαχειριστές και τους πελάτες (που δεν χρειάζεται
 να είναι οι ίδιοι).
 Τυπικά παραδείγματα των ενδιαφερομένων μερών λογισμικού περιλαμβάνουν
 (αλλά δεν περιορίζονται σε αυτά) τα εξής:
 
   * Χρήστες: Αυτή η ομάδα αποτελείται από αυτούς που θα λειτουργήσουν
   το λογισμικό.
-  Ως επί το πλείστον είναι μια ετερογενής ομάδα που αφορά ανθρώπους
+  Ως επί το πλείστων είναι μια ετερογενής ομάδα που αφορά ανθρώπους
```
Comment (dspinellis) [inline manuscript/1-SoftReq.txt]: Please keep the original one.
https://el.wiktionary.org/wiki/%CF%89%CF%82_%CE%B5%CF%80%CE%AF_%CF%84%CE%BF_%CF%80%CE%BB%CE%B5%CE%AF%CF%83%CF%84%CE%BF%CE%BD

File: manuscript/1-SoftReq.txt [hunk 2]
```diff
@@ -235,14 +235,14 @@
 Συνήθως υπάρχουν πολλοί άνθρωποι που εμπλέκονται, εκτός από τον ειδικό
 απαιτήσεων, καθένας από τους οποίους έχει μερίδιο στο λογισμικό.
 Τα ενδιαφερόμενα μέρη ποικίλλουν μεταξύ των έργων, περιλαμβάνουν όμως
-πάντα τους χρήστες / διαχειριστές και τους πελάτες (που δεν χρειάζεται
+πάντα τους χρήστες/ διαχειριστές και τους πελάτες (που δεν χρειάζεται
```
Comment (dspinellis) [inline manuscript/1-SoftReq.txt]: According to the Chicago Manual of Style (5.122) there should be no spaces in any of the two sides of the solidus.

File: manuscript/1-SoftReq.txt [hunk 3]
```diff
@@ -325,7 +325,7 @@
 μεταξύ του πεδίου των χρηστών του λογισμικού (και άλλων ενδιαφερομένων
 μερών) και τον τεχνικό κόσμο του μηχανικού λογισμικού.
 Μια σειρά από εσωτερικά συνεπή μοντέλα σε διαφορετικά επίπεδα αφαίρεσης
-διευκολύνουν την επικοινωνία μεταξύ των χρηστών του λογισμικού / ενδιαφερόμενων
+διευκολύνουν την επικοινωνία μεταξύ των χρηστών του λογισμικού/ ενδιαφερόμενων
```
Comment (dspinellis) [inline manuscript/1-SoftReq.txt]: ditto

File: manuscript/1-SoftReq.txt [hunk 4]
```diff
@@ -365,7 +365,7 @@
   της περιοχής.
   Πρέπει να προσδιορίζονται οι σχέσεις μεταξύ των σχετικών εννοιών εντός
   του πεδίου εφαρμογής.
-  * Ενδιαφερόμενα μέρη/«οι έχοντες λόγο» (βλ. παράγραφο [Φορείς
+  * Ενδιαφερόμενα μέρη/ «οι έχοντες λόγο» (βλ. παράγραφο [Φορείς
```
Comment (dspinellis) [inline manuscript/1-SoftReq.txt]: ditto (and for all changes).

File: manuscript/1-SoftReq.txt [hunk 5]
```diff
@@ -790,7 +787,7 @@ models), καταστατικά μοντέλα (state models), μοντέλα μ
 επίτευξη συμφωνίας μεταξύ των πελατών και των αναδόχων ή προμηθευτών
 (σε έργα με γνώμονα την αγορά, αυτοί οι ρόλοι μπορούν να αναπαρασταθούν
 από τα τμήματα μάρκετινγκ και ανάπτυξης) σε ό,τι το προϊόν λογισμικού
-προσδοκάται ή όχι να κάνει.
+προσδοκάτε ή όχι να κάνει.
```
Comment (dspinellis) [inline manuscript/1-SoftReq.txt]: Please keep passive voice.  There are no hints of active voice in the preceding paragraph.

File: manuscript/1-SoftReq.txt [hunk 6]
```diff
@@ -811,7 +808,7 @@ models), καταστατικά μοντέλα (state models), μοντέλα μ
 αλλά, όσον αφορά τον προσδιορισμό απαιτήσεων του λογισμικού, αυτό
 μπορεί να συμπληρωθεί με επίσημες ή ημιεπίσημες περιγραφές. Η επιλογή
 των κατάλληλων συμβολισμών επιτρέπει σε συγκεκριμένες απαιτήσεις και 
-πτυχές της αρχιτεκτονικής λογισμικού να περιγραφούν ακριβέστερα και
+πτυχές της αρχιτεκτονικής λογισμικού να περιγράφουν ακριβέστερα και
```
Comment (dspinellis) [inline manuscript/1-SoftReq.txt]: stet

File: manuscript/1-SoftReq.txt (hunk 1 above)
Comment (dspinellis) [inline manuscript/1-SoftReq.txt]: Ah, you reverted it in the next commit. Please merge them by rebasing to avoid the noise.

**Comment by Christina96** [comment]
Changed spaces between words, 2 words corrected.

**Pull request merged**